setup.py

```diff
@@ -0,0 +1,14 @@
+from setuptools import setup
+
+setup(name='State of the Universe',
+      version='0.0',
+      description='Generate an overview of the current state of the Universe!',
+      url='https://github.com/ojhall94/stateoftheuniverse',
+      install_requires=['astroquery',
```

Comment:
Should we pin/constrain versions of dependencies?

Reply:
Ultimately yes, but for now, maybe let's just use the most recent version of everything and then freeze things when we have the first version up and running?